common/ast/src/parser/expr/expr_visitor.rs

```diff
@@ -57,7 +57,11 @@ pub trait ExprVisitor: Sized + Send {
             Expr::Subquery(subquery) => self.visit_subquery(subquery),
             Expr::Function(function) => self.visit_function(function).await,
             Expr::TryCast { expr, data_type } => self.visit_try_cast(expr, data_type).await,
-            Expr::Cast { expr, data_type } => self.visit_cast(expr, data_type).await,
+            Expr::Cast {
+                expr,
+                data_type,
+                pg_style: _pg_style,
```

Comment:
Does the new version of sqlparser introduced `pg_style` params?

Reply:
https://github.com/datafuse-extras/sqlparser-rs/commit/b0571e8f91cac092fc8f8873e8babcc05fb91974

Yes in this commit.

Reply:
O, @b41sh should know it better :D